iBudget/Entity/models.py

```diff
@@ -0,0 +1,82 @@
+from django.db import models
+from django.contrib.auth.models import (
+    BaseUserManager, AbstractBaseUser
+)
+
+
+class CustomUser(AbstractBaseUser):
+    email = models.EmailField(max_length=50, unique=True)
```

Comment:
`AbstractBaseUser` should already have `email`, is not it?

Reply:
Yes, but if we use this feature we would rewrite 'create_user' method. In my humble opinion, it would be better to leave this code.